scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -107,5 +107,39 @@ Use the *rhel-tools* container with the following:
 
 See the link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/getting-started-with-containers/chapter-11-using-the-atomic-tools-container-image[RHEL Tools Container documentation] for more information.
 
+[[scaling-performance-debugging-using-oa-image]]
+== Debugging {product-title} Using the OpenShift-Ansible Image
+
+Red Hat distributes an https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[openshift-ansible image], with specific checks focused on detecting common deployment issues.
+Use the following checks to help detect common scaling performance problems:
+
+[[diagnostic-checks]]
+.Diagnostic Checks
+[options="header"]
+|===
+
+|Check Name |Purpose
+
+|`*etcd_imagedata_size*`
+|This check measures the total size of OpenShift image data in an etcd cluster. Fails if the calculated size exceeds a user-defined limit. If no limit is specified, this check will fail if the size of OpenShift image data amounts to `50%`
+or more of the currently used space in the etcd cluster.
+
+A failure from this check indicates that a significant amount of space in etcd is being taken up by OpenShift image data, which can eventually result in your etcd cluster crashing.
+
+A user-defined limit may be set by passing the variable `etcd_max_image_data_size_bytes=400000000` to the `openshift_health_checker` role.
+
+|`*etcd_traffic*`
+|This check detects higher-than-normal traffic on an Etcd host. Fails if a `journalctl` log entry with an Etcd sync duration warning is found.
+
+|===
+
+
+Use the *openshift-ansible* diagnostic checks with the following:
+
+----
+# docker run --rm -it openshift/openshift-ansible /bin/bash -c "ansible-playbook playbooks/common/openshift-checks/check.yml --become --become-user root"
```

Comment:
Why do we need "/bin/bash -c"? Do we need it?
The [ENTRYPOINT](https://github.com/sclorg/s2i-base-container/blob/master/Dockerfile#L80) should allow us to call ansible-playbook directly.